packages/launcher/src/renderer/app/UserAlertView.js

```diff
@@ -109,11 +115,65 @@ const getPermissionDescription = (key: string, input?: ?string): ?string => {
   return null
 }
 
+const Container = styled.View`
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  position: fixed;
+  background-color: transparent;
+  height: 100%;
+`
+
+const RequestContainer = styled.View`
+  background-color: ${props => props.theme.colors.GREY_DARK_3C};
```

Comment:
I don't think props should be used to import constants, it should be from the `colors` module directly.

Reply:
Well.. we've been using it everywhere since the beginning.. We pass the COLOR variables in `theme.js` to be used in styled components.

Reply:
That sounds quite inefficient to me, unless I'm missing something it means these functions need to be called on every render even though they return a constant. Theming should be about injecting variables, not constants.
Anyways it's fine no need to change anything in this PR, but it's something we should consider overall.